OJP/OJP_JourneySupport.xsd

```diff
@@ -88,15 +96,9 @@
 			<xs:documentation>Vehicle journey (not dated).</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element ref="JourneyRef"/>
-			<xs:element name="PublicCode" type="xs:normalizedString" minOccurs="0">
-				<xs:annotation>
-					<xs:documentation>Public code of the SERVICE JOURNEY (same meaning as in NeTEx). Examples: "512", "S8" or "Circle Line" or "ICE 488".</xs:documentation>
-				</xs:annotation>
-			</xs:element>
 			<xs:element name="ServiceSection" type="ServiceSectionStructure" minOccurs="1" maxOccurs="unbounded">
```

Comment:
I see a circularity - I see *ServiceSectionStructure* containing *StopSequenceIntervalGroup* and *StopSequenceIntervalGroup* containing a *ServiceSectionStructure*.

Reply:
@trurlurl  StopSequenceIntervalGroup does not conatin a ServiceSectionSttructure as far as I can tell.

![image](https://github.com/VDVde/OJP/assets/24227470/2ff5c696-6634-48a1-8a3f-a7f7ed81565b)

Reply:
I must have been hallucinating something ...